### PR TITLE
Expand judges

### PR DESCRIPTION
**Pull request by yaobviously**
## Description

Added a pickled list of judges names (900+) from the EOIR dataset and redefined the appellate judges list in the BIACase class in ocr.py to hopefully capture more Panel member names when processing PDFs.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: app/ocr.py [hunk 1]
```diff
@@ -117,54 +118,8 @@ class BIACase:
...
-        "Neal, David L.",
-        "Noferi, Mark",
-        "O'Connor, Blair",
-        "O'Herron, Margaret M.",
-        "O'Leary, Brian M.",
-        "Owen, Sirce E.",
-        "Pauley, Roger",
-        "Petty, Aaron R.",
-        "Pepper, S. Kathleen",
-        "RILEY, KEVIN W.",
```
Comment (andrewlee977) [inline app/ocr.py:160]: Should "RILEY, KEVIN W." be formatted this way? I could see some potential issues with querying down the road.

**Review by andrewlee977 — CHANGES_REQUESTED** [review]
Looks good, makes it a lot more organized and will allow for transparency when populating our ds_cases database. Would check on formatting for "RILEY, KEVIN W.", though.

File: app/ocr.py (hunk 1 above)
Comment (KanadeFuzen) [reply app/ocr.py:160]: 🤔 I agree with that as well, Andrew.

Comment (yaobviously) [reply app/ocr.py:160]:
```
    with open('./app/judge_names.pkl', 'rb') as j:
        appellate_panel_members = pickle.load(j)
```

`I` believe all of those lines were deleted and replaced by the loaded pickle file.

Comment (andrewlee977) [reply app/ocr.py:160]: Great👍🏼

**Review by andrewlee977 — APPROVED** [review]
Look great

**Review by minh14496 — APPROVED** [review]

**Review by KanadeFuzen — APPROVED** [review]
Looks perfect. Thanks for that clarification! 👍

File: app/ocr.py [hunk 2]
```diff
@@ -117,54 +118,8 @@ class BIACase:
...
-        "Rosen, Scott",
-        "Snow, Thomas G.",
-        "Swanwick, Daniel L.",
-        "Wendtland, Linda S.",
-        "Wetmore, David H.",
-        "Wilson, Earle B."
-    ]
+    with open('./app/judge_names.pkl', 'rb') as j:
+        appellate_panel_members = pickle.load(j)
```
[the remark this answers is not in the document]
Comment (yaobviously) [reply app/ocr.py:122]: Yes

**Pull request merged**